### PR TITLE
Fix Attitude Controller Sign

### PR DESCRIPTION
**Pull request by shihaocao**
# Fix Attitude Controller Sign + Small Ptest Changes

Standby now points at _something_, and small changes

### Summary of changes
- Moved fast boot to the setup_of_boot util, allows for user to set things after the fast_booter executes
- Inverted the sign in FSW to turn a desired torque applied to satellite to desired torque applied to wheels.
- Inverted corresponding checks in unit tests

### Testing
HOOTL'd

The below image shows the satellite switching from detumble, into standby, and then slewing rapidly to point

image

### CODE REVIEW

File: ptest/cases/utils.py [hunk 1]
```diff
@@ -304,10 +304,12 @@ def run_boot_sequence(self):
...
     def finished_boot(self):
-        if self.fast_boot:
-            self.flight_controller.write_state("pan.state", Enums.mission_states[self.desired_boot_state])
+        if self.finished:
             return True
-        elif self.finished:
+        elif self.fast_boot:
+            self.flight_controller.write_state("pan.state", Enums.mission_states[self.desired_boot_state])
+            self.logger.put(f"[TESTCASE] Fast Boot: Setting MissionMode to {self.desired_boot_state} state.")
+            self.finished = True
```
Comment (shihaocao) [inline ptest/cases/utils.py:312]: I switched the order of the evaluation of the if statements, and added a print statement.

**Review by tanishqaggarwal — APPROVED** [review]

File: ptest/cases/utils.py [hunk 2]
```diff
@@ -239,6 +239,11 @@ def setup_boot(self):
 
         self.logger.put(f"Waiting for the satellite to boot to {self.desired_boot_state}.")
 
+        if self.fast_boot:
+            self.flight_controller.write_state("pan.state", Enums.mission_states[self.desired_boot_state])
+            self.logger.put(f"[TESTCASE] Fast Boot: Setting MissionMode to {self.desired_boot_state} state.")
+            self.finished = True
+
```
Comment (shihaocao) [inline ptest/cases/utils.py:246]: Moved fastboot setting into within setup. This allows `setup_post_bootsetup logic` to execute AFTER `fast_boot` sets it to something else.

File: ptest/psim/simulation.py [hunk 3]
```diff
@@ -204,8 +204,9 @@ def configure(self):
         if self.mapping_file_name == "":
             raise RuntimeError("Error. Please set the json mapping file name property")
 
-        configs = [prefix + x + postfix for x in self.sim_configs]
-        self.mysim = psim.Simulation(self.sim_model, configs)
+        configs_list = [prefix + x + postfix for x in self.sim_configs]
+        config = psim.Configuration(configs_list)
+        self.mysim = psim.Simulation(self.sim_model, config)
```
Comment (shihaocao) [inline ptest/psim/simulation.py:209]: Hot fix to be compatible with latest psim. Actual configuration from ptest side will come later.

Comment (tanishqaggarwal) [reply ptest/psim/simulation.py:209]: I'll take your word for it, I have no idea what this syntax does LOL

**Review by tanishqaggarwal — APPROVED** [review]

File: ptest/psim/simulation.py (hunk 3 above)
Comment (kkrol27) [reply ptest/psim/simulation.py:209]: Okay lol I was gonna say the config will need to be exposed to individual cases to override things.

File: src/fsw/FCCode/AttitudeController.cpp [hunk 4]
```diff
@@ -223,6 +223,11 @@ void AttitudeController::calculate_pointing_controller() {
...
     // Call the controller and write results to appropriate state fields
     control_pointing(pointer_state, pointer_data, pointer_actuation);
+
+    // the output of GNC is a desired torque to the spacecraft, so we invert the command
+    // to obtain a torque to the wheels to effect the original command
+    pointer_actuation.rwa_body_cmd = -1*pointer_actuation.rwa_body_cmd;
+
     if (lin::all(lin::isfinite(pointer_actuation.mtr_body_cmd) && lin::isfinite(pointer_actuation.rwa_body_cmd))) {
         m_body_cmd_f.set(pointer_actuation.mtr_body_cmd);
         t_body_cmd_f.set(pointer_actuation.rwa_body_cmd);
```
Comment (kkrol27) [inline src/fsw/FCCode/AttitudeController.cpp:233]: This is alright for right now but writing the wheel command to `t_body_cmd` may be confusing field nomenclature wise.

**Review by kkrol27 — APPROVED** [review]
All looks good!

File: src/fsw/FCCode/AttitudeController.cpp (hunk 4 above)
Comment (shihaocao) [reply src/fsw/FCCode/AttitudeController.cpp:233]: ah good catch lmao... I'll fix it on the next one since this PR is a blocker for local work

**Pull request merged**